cumulusci/tasks/command.py

```diff
@@ -107,7 +107,6 @@ def _run_command(
             stdout=sys.stdout if interactive_mode else subprocess.PIPE,
             stderr=sys.stderr if interactive_mode else subprocess.PIPE,
             stdin=sys.stdin if interactive_mode else subprocess.PIPE,
-            bufsize=1,
```

Comment:
(The warning said that this option is unused because the pipes are in binary mode)